tests/unit/amo/components/TestAddonReviewCard.js

```diff
@@ -1052,6 +1052,18 @@ describe(__filename, () => {
       );
     });
 
+    it('renders a byLine without a link when the review addon has an empty slug', () => {
+      const review = signInAndDispatchSavedReview({
+        externalReview: {
+          ...fakeReview,
+          addon: { ...fakeReview.addon, slug: '' },
```

Comment:
It would be nice to put a comment here with a link to the bug. Otherwise, it seems weird that this test case exists, given the Flow type for `reviewAddon`.